cli/console.py

```diff
@@ -101,22 +104,76 @@ def inspect_task(
         sp_credential = obtain_sp_credential()
         full_blob_path = f"{job_id}/{task_filename}"
         try:
-            blob_service_client = instantiate_blob_service_client(
-                sp_credential=sp_credential,
-                account_url=azure_storage["azure_storage_account_url"],
+            blob_data = download_blob(
+                blob_path=full_blob_path, sp_credential=sp_credential
             )
-            blob_client = blob_service_client.get_blob_client(
-                container=azure_storage["azure_container_name"], blob=full_blob_path
-            )
-            downloader = blob_client.download_blob(max_concurrency=1, encoding="utf-8")
-            blob_text = downloader.readall()
-            console.print(json.loads(blob_text))
+            console.print(blob_data)
+
         except (ResourceNotFoundError, ValueError, ClientAuthenticationError) as e:
             console.print(
                 "[italic red] :triangular_flag: Error instantiating blob client or finding specified resource."
             )
             raise e
 
 
+@app.command("modify-task")
+def modify_task(
+    job_id: Annotated[
+        str, typer.Option("--job-id", "-j", help="Job ID to list tasks for")
+    ],
+    task_filename: Annotated[
+        str, typer.Option("--task-filename", "-t", help="Task filename to inspect")
+    ],
+):
```

Comment:
This is not something that should happen in this PR, but at some point we should allow the user to pass in storage container as an argument as well.